wordpress-mobile/WordPress-Android
# App settings

Fixes #3843. This PR separates local settings from WordPress.com settings into a new page.

@tonyr59h since you already reviewed all the Account Settings PRs so far, do you mind checking this out as well? Thanks!
@daniloercoli: I've added a new event for opened App Settings. Could you review the relevant section? Thanks!
@mattmiklic & @rickybanister: Here are the screenshots of the pages, could you check if they look correct? Thanks! https://cloudup.com/ckBirKCeor0

To test:
Login with your WordPress.com account and test out everything in Account Settings page.
Logout from WordPress.com account and make sure "Account Settings" is removed from the Me page.
Test everything in App Settings, especially changing language & pin lock feature.

Needs review: @tonyr59h
